src/app/cdk.ts

```diff
@@ -12,12 +12,12 @@ export interface IAppProps<ConfigT extends IConfig> {
  * You may provide a configuration to apply
  */
 export class JetKitCdkApp<ConfigT extends IConfig> extends CdkApp {
-  config: ConfigT
+  config?: ConfigT | undefined
```

Comment:
```suggestion
  config?: ConfigT
```

Reply:
Shouldn't this be enough?

Reply:
I thought so but TS was making a distinction between the element not being present in an object vs being undefined. It was odd to me. Happy to show you on a call

Reply:
<img width="711" alt="Screen Shot 2021-06-25 at 7 11 17 PM" src="https://user-images.githubusercontent.com/245131/123454351-203e9280-d5e9-11eb-82e9-bea14f3b6c44.png">

Lmk if you wanna chat about it, I'm confused